src/pages/DailyGardening/index.tsx

```diff
@@ -86,8 +87,8 @@ export const DailyGardening = () => {
   const completeTaskHandler = useCallback(
     async (rule: Rule) => {
       if (rule._id) {
-        setLastClicked(rule._id)
-      };
+        setLastClicked(rule._id);
+      }
```

Comment:
Not sure why it was deleted in my pr sorry I ended up adding it again🙏 cc: @conalli